openmeter/app/clickhouse.go

```diff
@@ -0,0 +1,20 @@
+package app
+
+import (
+	"fmt"
+
+	"github.com/ClickHouse/clickhouse-go/v2"
+	"github.com/ClickHouse/clickhouse-go/v2/lib/driver"
+
+	"github.com/openmeterio/openmeter/config"
+)
+
+// TODO: add closer function?
```

Comment:
Agreed, we need to gracefully close connections to server. Same way as we do for pgx/ent.